src/types.ts

```diff
@@ -128,6 +138,7 @@ export interface TxResult {
   events?: {
     [eventName: string]: EventLog;
   };
+  nonce?: number;
```

Comment:
This is only present in results passed through our internal sender? Can we mark it as such with a comment?